core/test-framework.cabal

```diff
@@ -52,7 +52,7 @@ Library
                               , containers     >= 0.1    && < 0.7
                               , regex-posix    >= 0.72   && < 0.96
                               , old-locale     >= 1.0    && < 1.1
-                              , time           >= 1.1.2  && < 1.9
+                              , time           >= 1.1.2  && < 2
```

Comment:
This is *not* a PVP compliant upper bound. See https://pvp.haskell.org/faq/#semver for more details.